src/components/lablup-terms-of-service.ts

```diff
@@ -102,6 +102,9 @@ export default class LablupTermsOfService extends LitElement {
           --mdc-list-item__primary-text: {
             height: 20px;
           };
```

Comment:
Same here. Let's apply this style definition only when `mwc-select` enables `FixedMenuPosition` attribute.